### PR TITLE
Translations 20180620

### LINKED ISSUE #208: Small pending issues with translations
the labels reported as 'fuzzy' in translation file are not translated in the UI for both DE and IT languages

### PR DESCRIPTION
**Pull request by cezio**
fix #208

### CODE REVIEW

File: ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po [hunk 1]
```diff
@@ -693,56 +685,46 @@ msgid "Add new Creator"
...
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:255
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:87
-#, fuzzy
 msgid "OthId Identifier"
 msgstr "anderen Bezeichner"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:256
-#, fuzzy
 msgid "OthId dataset identifier"
 msgstr "dataset identifier"
```
Comment (tdipisa) [inline ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po]: use **Andere ID des Datensatzes**

File: ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po [hunk 2]
```diff
@@ -693,56 +685,46 @@ msgid "Add new Creator"
...
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:260
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:90
-#, fuzzy
 msgid "OthId Organization"
 msgstr "Organisation E-Mail"
```
Comment (tdipisa) [inline ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po]: use **Andere ID der Organisation**

File: ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po [hunk 3]
```diff
@@ -693,56 +685,46 @@ msgid "Add new Creator"
...
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:261
-#, fuzzy
 msgid "OthId organization name"
 msgstr "Organisation E-Mail"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:265
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:93
-#, fuzzy
 msgid "OthId Organization ID"
 msgstr "Organisation E-Mail"
```
Comment (tdipisa) [inline ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po]: use **Andere ID der Organisation** 

Is this a duplication of the above one?

File: ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po [hunk 4]
```diff
@@ -693,56 +685,46 @@ msgid "Add new Creator"
...
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:266
-#, fuzzy
 msgid "OthId organization identifier"
 msgstr "Organisation Website-URL"
```
Comment (tdipisa) [inline ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po]: the same as reported above 

Are these translations used? If not please remove them.

File: ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po [hunk 5]
```diff
@@ -693,56 +685,46 @@ msgid "Add new Creator"
...
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:233
-#, fuzzy
 msgid "Add new Temporal Coverage"
 msgstr "zeitliche Abdeckung"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:255
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:87
-#, fuzzy
 msgid "OthId Identifier"
 msgstr "anderen Bezeichner"
```
Comment (tdipisa) [inline ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po]: use **Code**

File: ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po [hunk 6]
```diff
@@ -693,56 +685,46 @@ msgid "Add new Creator"
...
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:260
 #: ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html:90
-#, fuzzy
 msgid "OthId Organization"
 msgstr "Organisation E-Mail"
 
 #: ckanext/dcatapit/templates/macros/dcatapit_form_macros.html:261
-#, fuzzy
 msgid "OthId organization name"
 msgstr "Organisation E-Mail"
```
Comment (tdipisa) [inline ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po]: use **es: ISBN**

File: ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po (hunk 4 above)
Comment (cezio) [reply ckanext/dcatapit/i18n/de/LC_MESSAGES/ckanext-dcatapit.po]: They are in use. Lineid shows where:

https://github.com/geosolutions-it/ckanext-dcatapit/blob/master/ckanext/dcatapit/templates/macros/dcatapit_form_macros.html#L251-L272

which is displayed in alternate identifier fields:

image


Also, some of them are in use in displaying those values

https://github.com/geosolutions-it/ckanext-dcatapit/blob/master/ckanext/dcatapit/templates/package/snippets/dcatapit_additional_info.html#L84-L97

**Pull request merged**